woocommerce/woocommerce-ios
# [Mobile Payments] Redirect Simple Payments from Order navigation to HubMenu

Closes: #7440 

### Description

This PR changes the navigation flow when a merchant attempts to create a `Simple Payment` within the Orders screen. Rather than being redirected to the Simple Payments flow opener, now they're shown a notice that informs them that the feature has been moved to the app's Menu tab (HubMenu). Upon tapping on the `Got it!`button, they're redirected to the app's Menu tab.

### Changes

If the feature flag is active, we intercept the current `presentOrderCreationFlow` and show a `BottomSheetViewController` in our `AddOrderCoordinator`. This bottom sheet is an existing WordPressUI component that already implements the base of what we need for displaying a modal view on-screen.

Through this modal, as the Order screen is done in UIKit, we use a `UIHostingViewController` which wraps a new reusable SwiftUI `BottomSheetNoticeView` View. Behaviour and configuration for this is done via the model `NewSimplePaymentsNoticeViewModel`.


### Testing instructions
1. Go to Orders > Tap "+" > Tap "Simple Payments" > See the new bottom sheet appear.
2. Tap in "Got it!" > Confirm you're redirected to the HubMenu

### Screenshots

| Light | Dark |
|--|--|
| <img width=400 src="https://user-images.githubusercontent.com/3812076/184135355-a3b43be3-d9f9-4adc-93fb-698cce5bb62c.png" > | <img width=400 src="https://user-images.githubusercontent.com/3812076/184135376-5348c5a1-259a-4a4c-a48e-691edf251399.png"> |

https://user-images.githubusercontent.com/3812076/184136336-f024b89c-1ade-4ee6-9b99-dce3d02d18a9.mp4


---
